README.md

```diff
@@ -469,3 +471,50 @@ type RankingItem = {
 - `first`: Limit the number of results. Type: number.
 - `rarity`: Filter the results by the rarity. Possible values: `unique`, `mythic`, `legendary`, `epic`, `rare`, `uncommon`, `common`.
 - `category`: Filter the results by wearable category. Possible values: `eyebrows`,`eyes`,`facial_hair`,`hair`,`mouth`,`upper_body`,`lower_body`,`feet`,`earring`,`eyewear`,`hat`,`helmet`,`mask`,`tiara`,`top_head`, `skin`.
+
+## Prices
+
+**Endpoint**: `/v1/prices`
+
+**Type**:
+
+```ts
+Record<string, number>
+```
+
+**Query Params**:
+
+- `category`: Filter by `NFTCategory`. Possible values: `land`, `parcel`, `estate`, `wearable`, `emote` and `ens`.
+- `assetType`: Filter by `AssetType`. Possible values: `item` or `nft`.
+- `isWearableHead`: Only return results that their category is `wearable` and are part of the avatar's head. Type `boolean`.
+- `isWearableAccessory`: Only return results that their category is `wearable` and accessories (not a part of the body).
+- `isWearableSmart`: Only return smart wearables. Type `boolean`.
+- `wearableCategory`: Filter results by `WearableCategory`. Possible values: `eyebrows`,`eyes`,`facial_hair`,`hair`,`mouth`,`upper_body`,`lower_body`,`feet`,`earring`,`eyewear`,`hat`,`helmet`,`mask`,`tiara`,`top_head`, `skin`.
+- `wearableGender`: Filter results by `GenderFilterOption`. It supports multiple values by adding the query param multiple times. Possible values: `male`, `female`, `unisex`.
+- `emoteCategory`: Filter results by `EmoteCategory`. Possible values: `dance`, `stunt`, `greetings`, `fun`, `poses`, `reactions`, `horror`, `miscellaneous`.
+- `emoteGender`: Filter results by `GenderFilterOption`. It supports multiple values by adding the query param multiple times. Possible values: `male`, `female`, `unisex`.
+- `emotePlayMode`: Filter results by `EmotePlayMode`. It supports multiple values by adding the query param multiple times. Possible values: `simple`, `loop`
+- `contractAddress`: Filter results by contract address. It supports multiple values by adding the query param multiple times. Type: `address`.
+- `itemRarity`: Filter results by `Rarity`. It supports multiple values by adding the query param multiple times. Possible values: `unique`, `mythic`, `legendary`, `epic`, `rare`, `uncommon`, `common`.
+- `network`: Filter results by `Network`. Possible values: `ETHEREUM`, `MATIC`.
+
+## Stats
+
+**Endpoint**: `/v1/stats/:resource/:stat`
```

Comment:
I know so far the only resource is `estate`, but is the idea that other possible resources would be `wearable`, `emote`, `ens`, etc? because if so then maybe we could rename it to `category` which is the term we use for that specific enum.